Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitSettings.cpp

```diff
@@ -89,6 +89,13 @@ static void testWebKitSettings(Test*, gconstpointer)
     webkit_settings_set_javascript_can_open_windows_automatically(settings, TRUE);
     g_assert_true(webkit_settings_get_javascript_can_open_windows_automatically(settings));
 
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+    // Hyperlink auditing is deprecated and always enabled.
+    g_assert_true(webkit_settings_get_enable_hyperlink_auditing(settings));
+    webkit_settings_set_enable_hyperlink_auditing(settings, FALSE);
+    g_assert_true(webkit_settings_get_enable_hyperlink_auditing(settings));
+    ALLOW_DEPRECATED_DECLARATIONS_END
```

Comment:
If this is desired you should probably also restore the test for webkit_settings_get_enable_dns_prefetching.

Reply:
Done in https://github.com/WebKit/WebKit/pull/47405 🙃